### PR TITLE
Copy and paste styles

### PR DESCRIPTION
**Pull request by Fausto95**
Fixes https://github.com/excalidraw/excalidraw/issues/209

### CODE REVIEW

**Comment by Fausto95** [comment]
@jaredpalmer I changed the binding of the `shift` key to `ctrl` because <kbd>cmd+shift+c</kbd> opens the devtools which is a bit annoying.
Let me know if this is ok for ux

**Comment by jaredpalmer** [comment]
Hrmmm. how does this work on windows?

**Comment by Fausto95** [comment]
> Hrmmm. how does this work on windows?

According to https://github.com/excalidraw/excalidraw/pull/216/files, it wouldn't work, so I'll just use shift

**Comment by nanot1m** [comment]
Just for note 📝 Figma stores styles in memory rather then clipboard. So it has better browser support 😊

**Comment by Fausto95** [comment]
> Just for note 📝 Figma stores styles in memory rather then clipboard. So it has better browser support 😊

That's what I was thinking about, since the navigator.clipboard api is not in every browser.
I think I'll save it in memory

File: src/index.tsx [hunk 1]
```diff
@@ -212,6 +213,29 @@ class App extends React.Component<{}, AppState> {
       }
       this.forceUpdate();
       event.preventDefault();
+      // Copy Styles: Cmd-Shift-C
+    } else if (event.metaKey && event.shiftKey && event.code === "KeyC") {
+      const element = elements.find(el => el.isSelected);
+      if (element) {
```
Comment (vjeux) [inline src/index.tsx]: What should the behavior be for more than 1 element?

Comment (Fausto95) [reply src/index.tsx]: It'd take the first that satisfies the condition `isSelected`.
The approach I took was that we're able to copy styles from a single shape, but when pasting we can do it on multiple shapes.

**Comment by vjeux** [comment]
Now that we have a context menu, could you add it there too? :)

**Pull request merged**